adele-ticket-master-api/src/main/java/com/microservicesteam/adele/ticketmaster/model/BookedTicket.java

```diff
@@ -9,6 +9,7 @@
     String bookingId();
 
     @Override
+    @Value.Default
```

Comment:
This annontation is crutial to have it in the `Immutables*` class and as a result in the JSON representation.